src/components/EventUserTable.js

```diff
@@ -80,71 +196,154 @@ export class EventUserTable extends Component {
+                  <Select
+                    value={rowData.registrationStatus}
+                    onClick={event => changeRegistration(event, rowData)}
+                    style={{
+                      backgroundColor:
+                        rowData.registrationStatus === REGISTRATION_STATUS.CHECKED_IN
+                          ? '#54D26E'
+                          : rowData.registrationStatus === REGISTRATION_STATUS.WAITLISTED
+                            ? '#F7D055'
+                            : rowData.registrationStatus === REGISTRATION_STATUS.CANCELLED
+                              ? '#E15453'
+                              : '#FFF',
+                      paddingLeft: '10px'
+                    }}>
+                    <MenuItem value={REGISTRATION_STATUS.WAITLISTED}>Waitlisted</MenuItem>
+                    <MenuItem value={REGISTRATION_STATUS.CHECKED_IN}>Checked in</MenuItem>
+                    <MenuItem value={REGISTRATION_STATUS.REGISTERED}>Registered</MenuItem>
+                    <MenuItem value={REGISTRATION_STATUS.CANCELLED}>Cancelled</MenuItem>
+                  </Select>
+                </div>
+              )
+            }
+          ]}
+          data={this.state.rows}
+          // Configure options for the table
+          options={{
+            search: true,
+            draggable: false,
+            padding: 'dense',
+            pageSize: 15,
+            pageSizeOptions: [15, 50, 100],
+            actionsColumnIndex: 5,
+            exportButton: true,
+            headerStyle: {
+              fontWeight: 'bold'
+            },
+            rowStyle: rowData => ({
+
+            })
+          }}
+        />
+      </React.Fragment>
+
     );
   }
 }
+
+/**
+ * represents a statistic and shows a row of the stats with a dropdown for charts
+ */
+class Statistic extends React.Component {
+  constructor(props) {
+    super(props);
+    this.changeVisibility = this.changeVisibility.bind(this)
+    this.state = {
+      visibility: { visible: false, style: { display: 'none' } },
+    };
+  }
+
+  /*
+  changes the visibility of the stats of whichever div was selected
+  */
+  changeVisibility() {
+    const invisible = { visible: false, style: { display: 'none' } }
+    const visible = { visible: true, style: { display: 'flex', paddingBottom: '20px', paddingLeft: '50px' } }
+
+    if (this.state.visibility.visible) {
+      this.setState({ visibility: invisible })
+    } else {
+      this.setState({ visibility: visible })
+    }
+  }
+  render() {
+    const chartData = [
+      Object.keys(this.props.statObj).map(key => {
+        return {
+          label: key,
+          angle: this.props.statObj[key]
+        }
+      }),
+      Object.keys(this.props.statObj).map(key => {
+        return {
+          x: key,
+          y: this.props.statObj[key]
+        }
+      })
+    ]
+    return (
+      <React.Fragment>
+        <div style={styles.stats} onClick={this.changeVisibility}>
```

Comment:
You should use a `Paper` component from mui instead

Reply:
pretty cool, removes the need for that css border styling

Reply:
You might be able to apply those styles and click to the paper as well, but no biggie

Reply:
basically has the same affect, i like the Paper better because it extends the styling to the graphs when it becomes visible

Reply:
I meant the styles on the div could be put on the paper instead

Reply:
oh i dont think it works, the `display: flex` isn't showing the stats horizontally